inc/plugins/myalerts.php

```diff
@@ -822,11 +828,18 @@ function myalerts_global_intermediate()
 		$newAlertsIndicator = '';
 		if ($mybb->user['unreadAlerts']) {
 			$newAlertsIndicator = 'alerts--new';
+			if ($mybb->settings[$prefix.'headericon_visibility'] == "1")
+			{
+				$myalerts_headericon = eval($templates->render('myalerts_headericon'));
```

Comment:
`$myalerts_headericon` needs to be initialised to an empty value prior to this point to avoid PHP 8 warnings about unset variables.

In any case, my preferred approach is not to conditionally set this variable, but rather to conditionally set `$newAlertsIndicator`, to something like `alerts--no-new-so-hide`, and in the stylesheet style that class as `display: none;`.

I prefer it this way so that the Javascript can rely on this header item actually existing, even if it is hidden.